packages/card/docs.mdx

```diff
@@ -215,72 +122,34 @@ A component used to promote and link to marketing pages, always using a descript
   <Card
     link="https://hashicorp.com"
     thumbnail={{
-      src: "https://www.datocms-assets.com/19447/1648680074-screenshot-2022-03-31-at-00-40-47.png",
-      alt: "HashiConf Europe 2022 Recap"
+      src: 'https://www.datocms-assets.com/19447/1648680074-screenshot-2022-03-31-at-00-40-47.png',
+      alt: 'HashiConf Europe 2022 Recap',
     }}
-    meta={[
-      'USE CASE'
-    ]}
+    meta={['USE CASE']}
     heading="Display 5 - 4 lines with character limit of 130. Leo mauris fermentum pharetra blandit tellus euismod…"
-    productBadges={[
-        'waypoint',
-        'consul',
-      ]}
+    productBadges={['waypoint', 'consul']}
     description="Body small - 4 lines with character limit of 170. Leo mauris fermentum pharetra blandit tellus euismod pharetra. Duis egestas volutpat dolor…"
   />
   <Card
     link="https://hashicorp.com"
     thumbnail={{
-      src: "https://www.datocms-assets.com/19447/1648680074-screenshot-2022-03-31-at-00-40-47.png",
-      alt: "HashiConf Europe 2022 Recap"
+      src: 'https://www.datocms-assets.com/19447/1648680074-screenshot-2022-03-31-at-00-40-47.png',
+      alt: 'HashiConf Europe 2022 Recap',
     }}
-    meta={[
-      'USE CASE'
-    ]}
+    meta={['USE CASE']}
     heading="Display 5 - 4 lines with character limit of 130. Leo mauris fermentum pharetra blandit tellus euismod…"
-    productBadges={[
-        'terraform',
-        'consul',
-      ]}
+    productBadges={['terraform', 'consul']}
     description="Body small - 4 lines with character limit of 170. Leo mauris fermentum pharetra blandit tellus euismod pharetra. Duis egestas volutpat dolor…"
   />
   <Card
     link="https://hashicorp.com"
     thumbnail={{
-      src: "https://www.datocms-assets.com/19447/1648680074-screenshot-2022-03-31-at-00-40-47.png",
-      alt: "HashiConf Europe 2022 Recap"
+      src: 'https://www.datocms-assets.com/19447/1648680074-screenshot-2022-03-31-at-00-40-47.png',
+      alt: 'HashiConf Europe 2022 Recap',
     }}
-    meta={[
-      'USE CASE'
-    ]}
+    meta={['USE CASE']}
     heading="Display 5 - 4 lines with character limit of 130. Leo mauris fermentum pharetra blandit tellus euismod…"
-    productBadges={[
-      'nomad',
-      'vault',
-      'terraform'
-    ]}
+    productBadges={['nomad', 'vault', 'terraform']}
     description="Body small - 4 lines with character limit of 170. Leo mauris fermentum pharetra blandit tellus euismod pharetra. Duis egestas volutpat dolor…"
   />
 </div>
-
-## Base Props
-
-<PropsTable props={componentProps.baseProps} />
-
-## Sub-component-specific Props
-
-### `<Thumbnail />`
-
-<PropsTable props={componentProps.thumbnailProps} />
-
-### `<Meta />`
-
-<PropsTable props={componentProps.metaProps} />
-
-### `<Heading />`
-
-<PropsTable props={componentProps.headingProps} />
-
-### `<Description />`
-
-<PropsTable props={componentProps.descriptionProps} />
```

Comment:
Should these prop docs be removed?

Reply:
They were, I think your commenting on code that was deleted.

Reply:
Right, I'm asking if they _should_ be removed :+1: I was thinking we should have prop docs in this file for the component?

Reply:
ah I see what you mean, I can bring them back if you'd like. I though folks would not be interacting with these sub components outside of building the variants within react-components.

Reply:
ah yeah, good call! https://github.com/hashicorp/react-components/pull/810/commits/4deaaaf47bc92c5b6f282ea1be4877f14a2c36cf